Content.Server.Database/Model.cs

```diff
@@ -1119,4 +1120,16 @@ public class RoleWhitelist
         [Required]
         public string RoleId { get; set; } = default!;
     }
+
+    [Table("text_automod")]
+    public sealed class TextAutomod
+    {
+        [Column("id")] public int Id { get; set; }
+
+        public string Pattern { get; set; } = string.Empty;
```

Comment:
`string.Empty` is no more valid of a default value than `null!`. I'm pretty sure EF Core understands `required` properties now so you could use that, but if you don't feel like experimenting just keep using `null!`.

Reply:
Used `null!` instead of `string.Empty` [b8ac9e8](https://github.com/space-wizards/space-station-14/pull/28570/commits/b8ac9e8da860eaf7dd6c0834b973d5dbfb27944a)